### PR TITLE
Update co-operative.md

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: co-operative.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Hypha Worker Co-operative
 
-## About Us 
+## About Us
+
+Hypha is a worker co-operative based in Toronto who helps organizations and communities redesign their relationships with digital technology. Our team of technologists, designers, and community organizers provide digital strategy and coaching, research, web and mobile design and development, and infrastructure deployment services.
```
Comment (dcwalk) [inline co-operative.md:5]:
```suggestion
Hypha is a worker co-operative based in Toronto which helps organizations and communities redesign their relationships with digital technology. Our team of technologists, designers, and community organizers provide digital strategy and coaching, research, web and mobile design and development, and infrastructure deployment services.
```
technically "which" more correct if we refer to an entity not "human". But also -- curious why adding here? This copy looks like a mash up of our external-facing website, not sure I see it as necessary?

**Review by dcwalk — COMMENTED** [review]
no block, slight grammatical suggestion

File: co-operative.md (hunk 1 above)
Comment (patcon) [reply co-operative.md:5]: Heh thx :)

And as for why here, guess it was just where i came for this sorta of pull quote when Ben asked. Open to suggestions 

Options coming to mind:
1. Here
2. elsewhere in handbook
3. Press kit/page
4. Brand guide ([offering deck](https://docs.google.com/presentation/d/1y8Mnyw147rsX1n5NsDek6dJHl9PWU3cwG-_QXBMfY98/edit?usp=drivesdk) is closest)
5. Nowhere (keep it in each doc/proposal)

**Review by patcon — APPROVED** [review]
Lgtm!

File: co-operative.md (hunk 1 above)
Comment (benhylau) [reply co-operative.md]: It's the copy we are floating around to all our projects. Having it here is a good place to copy from. Probably when ppl come to our handbook this is more relevant info than our org structure.

**Pull request merged**